modules/attributes/pages/positional-and-named-attributes.adoc

```diff
@@ -56,3 +58,22 @@ No special processing is performed, except for attribute reference expansion, if
 
 If the attribute value contains the same quote character being used to enclose the value, escape the quote character in the value by prefixing it with a backslash (e.g., `title="\"Dark Horse\" is a song by George Harrison"`).
 // end::subs[]
+
+== Attribute parsing
+
+The exact rules for determining the boundaries of attributes and whether an attribute is positional or named are somewhat complicated.
+
+A `name` consists of a word character (letter or numeral) followed by any number of word characters, `.` charactes, or `-` characters, followed by something else (including the end of the string)
+
+* Parsing an attribute proceeds from the beginning of the attribute list string or after a previously recognized delimiter (`,`).
+* Blank space is skipped.
+* If there is a name, followed by optional blank space, followed by `=`, then a named attribute is recognized, with the just-read name.
+Parsing the value proceeds after skipping blank space after the `=`.
+* Otherwise this is a positional attribute with value starting after the skipped blank space and ending before (optional) blank space preceding the first delimiter (`,`).
+
+* Attribute value parsing, after skipping initial blank space:
```

Comment:
Drop the "after skipping the initial blank space". We have a dedicated sentence that addresses the optional space around the equals sign.